preview/README.md

```diff
@@ -0,0 +1,10 @@
+.NET CLI Preview Docker Image
```

Comment:
That's a good point. I was also thinking that the title of the README in the dotnet-docker repo should be updated; currently it also says ".NET CLI Preview Docker Image".  I'm thinking just calling it ".NET CLI Docker Image" would be better.